build.rs

```diff
@@ -370,6 +464,45 @@ fn run_command_with_args<S>(command_name: S, args: &Vec<String>)
     }
 }
 
+fn make_asm(source: &str, mut dst: PathBuf, target_info: &TargetInfo)
+            -> String {
+    let p = Path::new(source);
```

Comment:
Here we need to consider `crypto/perlasm/*.pl` and `crypto/sha/{sha-armv8.pl, sha-x86_64.pl}` the same way we consider *.h and *.inl in the `compile()` function.